### PR TITLE
Update v0.2.0 spec changes

### PR DESCRIPTION
**Pull request by frenchi**
This PR primarily addresses https://github.com/openvex/examples/issues/2  - the changes introduced with v0.2.0 spec.

Also it wasn't clear why the code examples were compressed in `hello-vex.zip`, so this PR:
-  unzips the example go files and to include them directly in the repo, and
- fixes a couple of typos.

### CODE REVIEW

**Review by cpanato — DISMISSED** [review]
it lgtm
but I will defer to @puerco for final review

File: code/hello/README.md [hunk 1]
```diff
@@ -20,21 +20,27 @@ func main() {
 			Context:    vex.Context,
 			Author:     "John Doe",
 			AuthorRole: "VEXing Engineer",
-			Version:    "1.0",
+			Version:    1.0,
```
Comment (puerco) [inline code/hello/README.md]: This is an int so better
```suggestion
			Version:    1,
```

File: code/hello/README.md [hunk 2]
```diff
@@ -20,21 +20,27 @@ func main() {
...
+			ID:          "CVE-2014-123456",
+			Description: "It's really bad",
+		},
+		Products: []vex.Product{
+			{
+				Component:     vex.Component{ID: "pkg:generic/1.0.0"},
+				Subcomponents: []vex.Subcomponent{},
+			},
+		},
+		Status:      "fixed",
```
Comment (puerco) [inline code/hello/README.md]:
```suggestion
		Status:      vex.StatusFixed,
```

File: code/hello/main.go [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+			ID:          "CVE-2014-123456",
+			Description: "It's really bad",
+		},
+		Products: []vex.Product{
+			{
+				Component:     vex.Component{ID: "pkg:generic/1.0.0"},
+				Subcomponents: []vex.Subcomponent{},
+			},
+		},
+		Status:      "fixed",
```
Comment (puerco) [inline code/hello/main.go]:
```suggestion
		Status:      vex.StatusFixed,
```

File: tutorials/tutorial02-merge.md [hunk 4]
```diff
@@ -3,148 +3,171 @@
...
 ## Merging with `vexctl`
 
 To combine documents, `vexctl` has a `merge` subcommand. The invocation is 
 simple enough: when running it, simply pass all the documents you want to
-merge and it will combine lla documents into one. This example can be run 
-within the vexctl repo. 
+merge and it will combine all documents into one.
 
-Lets inspect the contents of the documents:
+Let's generate inspect the contents of some example documents:
```
Comment (puerco) [inline tutorials/tutorial02-merge.md]: inspect or generate? :)

**Review by puerco — COMMENTED** [review]
thanks for updating these @frenchi I found a few nits but overall lgtm :)

File: tutorials/tutorial02-merge.md (hunk 4 above)
Comment (frenchi) [reply tutorials/tutorial02-merge.md]: derp, fixing now.

**Comment by frenchi** [comment]
nits fixed, should be good for another review, sorry for the back and forth @puerco, this was meant to be an easy merge 😅

**Review by puerco — APPROVED** [review]
Ah no problem at all, I'm happy to review as many as you want :D

Thanks a lot!!
/lgtm

**Review by puerco — APPROVED** [review]
Ah no problem at all, I'm happy to review as many as you want :D

Thanks a lot!!
/lgtm

**Pull request merged**